apps/rtm/index.js

```diff
@@ -1,27 +1,29 @@
 'use strict';
 
-var hof = require('hof');
-var wizard = hof.wizard;
-var mixins = hof.mixins;
-var i18nFuture = hof.i18n;
-var router = require('express').Router();
-var path = require('path');
-var _ = require('underscore');
-var BaseController = require('hof').controllers.base;
+const hof = require('hof');
+const wizard = hof.wizard;
+const mixins = hof.mixins;
+const i18nFuture = hof.i18n;
+const BaseController = hof.controllers.base;
+const deepTranslate = hof.middleware.deepTranslate;
+const router = require('express').Router();
+const path = require('path');
+const _ = require('lodash');
 
-var fields = _.extend({}, require('./fields/'));
-var i18n = i18nFuture({
+const fields = _.cloneDeep(require('./fields/'));
+const i18n = i18nFuture({
   path: path.resolve(__dirname, './translations/__lng__/__ns__.json')
 });
 
-router.use(mixins(fields, {
+router.use(deepTranslate({
```

Comment:
This middleware is for the conditional translations, there currently aren't any in RTM. Should we use it anyway or only use it in apps that need it? @JoeChapman @bendiggle @gxxm discuss!